mix.exs

```diff
@@ -2,7 +2,7 @@ defmodule OAuth2.Mixfile do
   use Mix.Project
 
   @source_url "https://github.com/scrogson/oauth2"
-  @version "2.0.1"
+  @version "3.0.0"
```

Comment:
```suggestion
  @version "2.1.0"
```

Do you think it would be a breaking change or more like a feature? Thoughts?

Reply:
I don't think this is a breaking change in a way but since the 1.0 -> 2.0 wasn't a breaking change but just being safe I think it's better to play it safe.

Reply:
What I'm worried about if people have made hackney specific changes and delete their mix.lock file

Reply:
I am gonna make the call to use `2.1.0`, just want to avoid a major release version when it isn't a breaking change.

Hopefully, I am not wrong on this one.